torchci/components/CommitStatus.tsx

```diff
@@ -92,6 +92,7 @@ export default function CommitStatus({
           !isRerunDisabledTestsJob(job) &&
           !isUnstableJob(job)
         }
+        // pred={(job) => job.conclusion === "pending"}
```

Comment:
Commented out?

Reply:
Thanks for the catch, yeah it's for debugging